src/main/java/io/reactiverse/pgclient/PgClient.java

```diff
@@ -55,6 +57,23 @@ static PgPool pool(Vertx vertx, PgPoolOptions options) {
     return new PgPoolImpl(vertx, false, options);
   }
 
+  /**
+   * Create a connection pool to the database configured with the given {@code connectionUri}.
+   *
+   * @param connectionUri the connection URI
+   * @return the connection pool
+   */
+  static PgPool pool(String connectionUri) {
```

Comment:
we should not need both for the moment as user should do it itself using the provider